metrics/collector.go

```diff
@@ -112,6 +112,7 @@ func (c Collector) Collect(ch chan<- prometheus.Metric) {
 }
 
 func updateMachineStatus(ctx context.Context, model *sabakan.Model) error {
+	MachineStatus.Reset()
```

Comment:
This function may be called concurrently.
In rare cases, the metrics may be empty.

The documentations said this function must be implemented in a concurrency safe way.
https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Collector

Use mutex to guard Collect method.